com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ManifestBuilder.java

```diff
@@ -402,18 +201,21 @@ public void addSupportedEngineVersion(String version) {
         }
     }
 
+    public void addResourceEntry(String url, HashDigest hash, int flags) throws IOException {
```

Comment:
We already have a hash digest when calling this function from ArchiveBuilder so no need to calculate it again.